src/Template/Layout/js/app/components/relation-view/relations-add.js

```diff
@@ -41,18 +41,49 @@ export default {
             type: String,
             default: '[]',
         },
+        csrfToken: {
+            type: String,
+            required: true,
+        } // csfrToken used for api call
     },
+
     data() {
         return {
             method: 'relationshipsJson',
             endpoint: '',
+            loading: false,
             selectedObjects: [],
-
             activeFilter: {},
-            loading: false,
+
+            // create object form data
+            saving: false,
+            showCreateObjectForm: false,
+            file: null,
+            objectType: '',
+            titlePlaceholder: 'title',
         };
     },
 
+    computed: {
+        /**
+         * return accepted mimetypes
+         *
+         * @return {String} mimetypes
+         */
+        acceptedMimeTypes() {
+            return 'audio/*,video/*,image/*,.pdf';
```

Comment:
it would be nice have a config for accepted mime types in `app.php` or elsewhere

Reply:
The API should tell us about this. Since we currently don't have that informtion I'd prefer to accept everything for now.